tests/hypertext/HTMLLintBearTest.py

```diff
@@ -9,21 +9,47 @@
 </html>
 """
 
+invalid_test = """
```

Comment:
could you maybe name it a bit more precise? Usually it's always: Give a name that resembles the usage. Like `file_with_invalid_spacing` or so